mlflow-v2-examples/kserve-example.ipynb

```diff
@@ -0,0 +1,440 @@
+    "            e,\n",
+    "        )\n",
+    "\n",
+    "    # Split the data into training and test sets. (0.75, 0.25) split.\n",
+    "    train, test = train_test_split(data)\n",
+    "\n",
+    "    # The predicted column is \"quality\" which is a scalar from [3, 9]\n",
+    "    train_x = train.drop([\"quality\"], axis=1)\n",
+    "    test_x = test.drop([\"quality\"], axis=1)\n",
+    "    train_y = train[[\"quality\"]]\n",
+    "    test_y = test[[\"quality\"]]\n",
+    "\n",
+    "    alpha = 0.5\n",
+    "    l1_ratio = 0.5\n",
+    "\n",
+    "    # create bucket\n",
+    "    object_storage = boto3.client(\n",
+    "        \"s3\",\n",
+    "        endpoint_url=os.getenv(\"MLFLOW_S3_ENDPOINT_URL\"),\n",
+    "        config=boto3.session.Config(signature_version=\"s3v4\"),\n",
+    "    )\n",
+    "    default_bucket_name = \"mlflow\"\n",
+    "\n",
+    "    buckets_response = object_storage.list_buckets()\n",
+    "    result = [\n",
+    "        bucket\n",
+    "        for bucket in buckets_response[\"Buckets\"]\n",
+    "        if bucket[\"Name\"] == default_bucket_name\n",
+    "    ]\n",
+    "    if not result:\n",
+    "        object_storage.create_bucket(Bucket=default_bucket_name)\n",
+    "\n",
+    "    with mlflow.start_run():\n",
+    "        lr = ElasticNet(alpha=alpha, l1_ratio=l1_ratio, random_state=42)\n",
+    "        lr.fit(train_x, train_y)\n",
+    "\n",
+    "        predicted_qualities = lr.predict(test_x)\n",
+    "\n",
+    "        (rmse, mae, r2) = eval_metrics(test_y, predicted_qualities)\n",
+    "\n",
+    "        print(\"Elasticnet model (alpha=%f, l1_ratio=%f):\" % (alpha, l1_ratio))\n",
+    "        print(\"  RMSE: %s\" % rmse)\n",
+    "        print(\"  MAE: %s\" % mae)\n",
+    "        print(\"  R2: %s\" % r2)\n",
+    "\n",
+    "        mlflow.log_param(\"alpha\", alpha)\n",
+    "        mlflow.log_param(\"l1_ratio\", l1_ratio)\n",
+    "        mlflow.log_metric(\"rmse\", rmse)\n",
+    "        mlflow.log_metric(\"r2\", r2)\n",
+    "        mlflow.log_metric(\"mae\", mae)\n",
+    "\n",
+    "        tracking_url_type_store = urlparse(mlflow.get_tracking_uri()).scheme\n",
+    "\n",
+    "        result = mlflow.sklearn.log_model(\n",
+    "            lr, \"model\", registered_model_name=\"ElasticnetWineModel2\"\n",
+    "        )\n",
+    "        model_uri = f\"{mlflow.get_artifact_uri()}/{result.artifact_path}\"\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d9e27a59-e600-4012-b2b5-f0844d919332",
+   "metadata": {},
+   "source": [
+    "Now you can visit the mlflow UI to see the model being registered. MLflow ui is running on Nodeport 31380 (by default, you can always inspect with `kubectl get svc -n kubeflow`). If you are running this on VM you can tunnel the port of the nodeport to your localhost with `ssh -i <keypair> -L 31380:localhost:31380 <user>@vm-ip>`. Afther that you can visit http://localhost:31380/.\n",
```

Comment:
some typos
```suggestion
    "Now you can visit the MLflow UI to see the model being registered. MLflow UI is running on Nodeport 31380 (by default, you can always inspect with `kubectl get svc -n kubeflow`). If you are running this on VM you can tunnel the port of the nodeport to your localhost with `ssh -i <keypair> -L 31380:localhost:31380 <user>@vm-ip>`. Afther that you can visit http://localhost:31380/.\n",
```

Reply:
Also, while those instructions are needed, they are a bit mixed with the results from the previous steps. Maybe a heading or sth to separate those two would help here too.